lua/modules/item_grid.lua

```diff
@@ -678,16 +678,17 @@ itemGrid.create = function(_, config)
 		if self.setGridEntries ~= nil then
 			self:setGridEntries({})
 		end
-
 		if config.type == "items" then
-			local req = api:getItems({
+			local reqConfig = {
 				minBlock = config.minBlocks,
 				repo = config.repo,
 				category = config.categories,
 				page = 1,
 				perpage = 250,
 				search = search,
-			}, function(err, items)
+				useCache = false,
```

Comment:
I think this turns off the cache for all requests (even when not in "create" mode). 
A `useCache` field could be added to the config received by `itemGrid.create = function(_, config)`
That would allow to indicate upstream if we want the cache to be ON or not for the grid.